torch_sim/runners.py

```diff
@@ -491,7 +491,7 @@ class StaticState(type(state)):
 
     trajectory_reporter.finish()
 
-    if isinstance(batch_iterator, ChunkingAutoBatcher):
+    if isinstance(batch_iterator, BinningAutoBatcher):
```

Comment:
while we're at it, maybe rename `batch_iterator`->`auto_batcher`? and `_configure_batches_iterator`-> `_configure_auto_batcher`

Reply:
The naming is different because if autobatcher is `False` then `_configure_auto_batcher` returns a list instead of an autobatcher. Maybe it's more confusing this way though?

Reply:
Done, also fixed the typing, good catch!